Cargo.toml

```diff
@@ -28,6 +28,7 @@ btf-rs = "1.0"
 byteorder = "1.5"
 caps = "0.5"
 clap = { version = "4.0", features = ["derive", "string"] }
+clap_complete = { version = "4.4.6", features = ["unstable-dynamic"] }
```

Comment:
nit: Please use `x.y` numbers only in version (`4.4`).

Reply:
Got it